app/forms/journeys/further_education_payments/have_one_login_account_form.rb

```diff
@@ -0,0 +1,37 @@
+module Journeys
+  module FurtherEducationPayments
+    class HaveOneLoginAccountForm < Form
+      attribute :have_one_login_account, :string
+
+      validates :have_one_login_account,
+        inclusion: {
+          in: ["true", "false", "i_dont_know"],
```

Comment:
Would be tempted to change this to 
```suggestion
          in: ["yes", "no", "i_dont_know"],
```
"true" and "false" as strings seem like they might trip us up in future